internal/workflow_testsuite_test.go

```diff
@@ -123,7 +123,7 @@ func TestUnregisteredActivity(t *testing.T) {
 	require.True(t, errors.As(err, &activityErr))
 
 	err = errors.Unwrap(activityErr)
-	var err1 *PanicError
```

Comment:
Any custom error results in being wrapped up within a `PanicError`?  I'm surprised to see this as we were seeing different behavior in the case of real invocation where the activity fails silently and workflow gets stuck.  Looks like the behavior in the case of testEnv differs from real execution.

Reply:
Application error [is the default](https://github.com/temporalio/sdk-go/blob/a39bbe82b2bab1281d24d7f18a93a9813e0e622b/internal/error.go#L889).